src/components/utils.js

```diff
@@ -18,4 +18,12 @@ export const getSingleArticle = (article_id) => {
     .then(({data})=>{
         return data.article;
     })
-};
\ No newline at end of file
+};
+
+export const getComments = (article_id) =>{
+    return newsApi
+    .get(`https://be-news-project.onrender.com/api/articles/${article_id}/comments`)
```

Comment:
as you've got a baseURL you shouldn't need to pass your `get` the first part of the URL, only the relevant endpoints

Reply:
(this goes for all your api functions)